_pages/resources.md

```diff
@@ -10,8 +11,7 @@ redirect_from:
     - /ta_titleiii.html
 ---
 
-{% include ta-link-back.html %}
-
+{% include alert.html type="info" text="The complete inventory of our ADA Resources is coming soon to this page. In the meantime, you can access them on our former Technical Assistance Materials page." %}
```

Comment:
can you explain this part of the change? I don't see this in the PR description. Should you remove ta-link-back.html if this replaces it?

Reply:
This replaces the ta-link-back.html so I suppose since it's not used anywhere else we can just delete the file.

Reply:
yeah let's delete it. If we need it in the future it'll be in the commit history.